src/chainstate/stacks/db/mod.rs

```diff
@@ -489,9 +489,10 @@ const STACKS_CHAIN_STATE_SQL: &'static [&'static str] = &[
         block_size TEXT NOT NULL,       -- converted to/from u64
 
         PRIMARY KEY(consensus_hash,block_hash)
-    );
-    CREATE UNIQUE INDEX index_block_hash_to_primary_key(index_block_hash,consensus_hash,block_hash);
-    "#,
+    );"#,
+    "CREATE UNIQUE INDEX index_block_hash_to_primary_key ON block_headers(index_block_hash,consensus_hash,block_hash);",
```

Comment:
We could safely drop the `UNIQUE` constraint here, because `(consensus_hash, block_hash)` is unique constrained and so is `index_block_hash`, and then this index is identical to `block_index_hash_index` below, so one of them should be dropped.

We _do_ query directly by `index_block_hash`, though, so it may be worthwhile to create an index on _just_ `index_block_hash`.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2420/commits/85822097eb7f13969a83f4d096df208b50896db7